spec/jobs/in_person/send_proofing_notification_job_spec.rb

```diff
@@ -145,28 +146,73 @@
           end
         end
 
-        it 'sends a message that respects the user email locale preference' do
-          allow(Telephony).to receive(:send_notification).and_return(sms_success_response)
+        context 'sends a message that respects the user email locale preference' do
+          let(:proofed_date) { Time.zone.now.strftime('%m/%d/%Y') }
+          let(:phone_number) { passed_enrollment.notification_phone_configuration.formatted_phone }
+          let(:contact_number) { '(844) 555-5555' }
+          let(:reference_string) { passed_enrollment.enrollment_code }
 
-          passed_enrollment.user.update!(email_language: 'fr')
-          passed_enrollment.update!(proofed_at: Time.zone.now)
-          proofed_date = Time.zone.now.strftime('%m/%d/%Y')
-          phone_number = passed_enrollment.notification_phone_configuration.formatted_phone
-          contact_number = '(844) 555-5555'
+          before do
+            allow(Telephony).to receive(:send_notification).and_return(sms_success_response)
+          end
 
-          expect(Telephony).
-            to(
-              receive(:send_notification).
-              with(
-                to: phone_number,
-                message: "Login.gov: Vous avez visité le bureau de poste le #{proofed_date}." \
-                         " Vérifiez votre e-mail pour votre résultat. Ce n'est pas vous?" \
-                          " Signalez-le immédiatement: #{contact_number}",
-                country_code: Phonelib.parse(phone_number).country,
-              ),
-            )
+          it 'handles English language preference' do
+            passed_enrollment.user.update!(email_language: 'en')
+            passed_enrollment.update!(proofed_at: Time.zone.now)
 
-          job.perform(passed_enrollment.id)
+            expect(Telephony).
+              to(
+                receive(:send_notification).
+                  with(
+                    to: phone_number,
+                    message: "Login.gov: You visited the Post Office on #{proofed_date}." \
+                      " Check email for your result." \
+                      " Not you? Report this right away: #{contact_number}." \
+                      " Ref: #{reference_string}",
```

Comment:
Should we format this code if we're expecting the user to read it out to someone like a support agent?

Elsewhere we use `Idv::InPerson::EnrollmentCodeFormatter` to add dashes, should we use that?

Reply:
@aduth I'm meeting today with @rutvigupta-design to double check the formatting of this field and that an abbreviation is ok. Since it's 16 digits dashes would be wise. I'm curious about how that'll play with the tight character limits of SMS, but I'll bring this up with Rutvi! Thanks

Reply:
That's a good point about SMS length. 👍

Reply:
@aduth should be handled in latest two commits. Thanks for calling out `EnrollmentCodeFormatter`